application/event_processor/change_event_validation.py

```diff
@@ -40,10 +40,8 @@ def check_ods_code_length(odscode: str, service_type: str) -> None:
         if len(odscode) != PHARMACY_ODSCODE_LENGTH:
             raise ValidationException(f"ODSCode Wrong Length, '{odscode}' is not length {PHARMACY_ODSCODE_LENGTH}.")
     if service_type == DENTIST_SERVICE_KEY:
```

Comment:
Can this function be condensed to take in a max ods code length value to compare against